setup.sh

```diff
@@ -29,7 +29,7 @@ if [ ! -f $CS/$LJ ]; then
     echo "
 Fetching launcher...
 "
-    curl "http://typesafe.artifactoryonline.com/typesafe/ivy-releases/org.scala-sbt/sbt-launch/$LJV/sbt-launch.jar" \
+    curl -L "http://dl.bintray.com/typesafe/ivy-releases/org.scala-sbt/sbt-launch/$LJV/sbt-launch.jar" \
```

Comment:
`https://repo.typesafe.com/typesafe/ivy-releases/org.scala-sbt/sbt-launch/X.Y.Z/sbt-launch.jar` should be used instead of hardcoding to Bintray to avoid future breakage.